rkt/fetch_test.go

```diff
@@ -35,6 +35,7 @@ import (
 	"github.com/coreos/rkt/pkg/keystore"
 	"github.com/coreos/rkt/pkg/keystore/keystoretest"
 	"github.com/coreos/rkt/rkt/config"
+	"github.com/coreos/rkt/rkt/rktflag"
```

Comment:
There is no such thing like "too much marketing". ;)

At first I wanted to name the package `flag`, but then I thought it would unnecessarily conflict with the standard `flag` package. But right now I realized, that we use `pflag` instead. Anyway, importing this package as `rktflag` is a good idea. Will rename it.